### ISSUE TITLE
Hyperparameter settings for multiple clients in fed average

### ISSUE THREAD

**Issue by Rahn80643** [JonasGeiping/invertinggradients#12]
Thank you for providing this awesome repository!

I have some questions about training with the settings federated average:

1.  If I set accumulation > 0  in inversefed/options.py and rec_multi.py, will this parameter set the number of clients in federated learning?
2.  Function train() in training_routine() seems doesn't include the part of training multiple clients and perform model aggregation in the server, may I ask is the implementation locates in another file?

Thank you

**Comment by JonasGeiping** [comment]
Hi, `accumulation` will set the number of local gradient steps contained in the update. The updated code at (https://github.com/JonasGeiping/breaching) is a bit simpler to use to set up multi-user aggregation (e.g. see here: https://github.com/JonasGeiping/breaching/blob/main/breaching/config/case/user/multiuser_aggregate.yaml) 


train() is only used to train a neural network (and used to load trained CIFAR-10 models). FL Training is not simulated in this repository, please check on the repositories for training with FL (such as flower) to simulate training with multiple clients.

**Comment by Rahn80643** [comment]
Thank you for the quick reply, I will tested on the updated code for the FL training
